src/Core/Content/Product/Cms/ProductSliderCmsElementResolver.php

```diff
@@ -59,7 +59,6 @@ public function collect(CmsSlotEntity $slot, ResolverContext $resolverContext):
 
         if ($products->isStatic() && $products->getValue()) {
             $criteria = new Criteria($products->getArrayValue());
-            $criteria->addAssociation('cover');
```

Comment:
why did you remove the cover? :thinking:

Reply:
@mitelg Because we don't need it here. It's already present by https://github.com/shopware/shopware/blob/trunk/src/Core/Content/Product/SalesChannel/SalesChannelProductDefinition.php#L60 and will be overriden by this anyway.

Reply:
ah okay, then it is okay. just want to make sure, that no data is missing after that change, because that would be breaking